senior-knights/course-schedulizer
# Import csv by url

To Test:
- Visit the app, but append something like `?xlsx=https://raw.githubusercontent.com/senior-knights/course-schedulizer/develop/client-course-schedulizer/csv/math-schedule.xlsx` or `?csv=https://raw.githubusercontent.com/senior-knights/course-schedulizer/develop/client-course-schedulizer/csv/math-schedule-full.csv` to the end of the url.
- It should load the schedule from that url.

Notes:
- 404s will be displayed in the console.
- I didn't worry too much about invalid files, but I think we can expect users to provide valid files (or notice their own error when not)? It seems that providing a csv when you say it's an xlsx or vice versa might actually still work based on my testing.
- The file in the url will only load if the initial state has an empty schedule AND the file should only load once on page refresh (try importing a different schedule after page load, it should work as expected). This is to prevent user work from being overwritten, but I can see how it might be confusing that the file provided in the url doesn't load if you have an existing schedule loaded, thoughts on this are appreciated.
